Homework_2.0.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, name, position):
 
     def check_ascii(self, param):
         if type(param) == str:
```

Comment:
To check the type of the object it would be better to use `isinstance` built-in function:
```
    if isinstance(param, str) and param.isascii():